prowler/lib/check/checks_loader.py

```diff
@@ -111,7 +111,7 @@ def load_checks_to_execute(
             ):
                 checks_to_execute.add(check_name)
         # Only execute threat detection checks if threat-detection category is set
-        if not categories or "threat-detection" not in categories:
+        if (not categories or "threat-detection" not in categories) and not check_list:
```

Comment:
Could you create a test if there is not to cover this use case in the `load_checks_to_execute`?

Reply:
Done!